Source/core/maps/MapQuestMapsManager.js

```diff
@@ -26,9 +26,12 @@ MapQuestMapsManager.prototype.getDistanceString = function () {
     var routingSummary = document.querySelector('.route-selection .distance');
     if(routingSummary) {
         var distanceString = routingSummary.innerText;
+        console.log('distanceString: ' + distanceString);
+        return distanceString;
     }
-    console.log('distanceString: ' + distanceString);
-    return distanceString;
+    console.error('no routingSummary found distanceString is 0');
+    
+    return 0;
```

Comment:
In case of success, we are returning a string (in line 30 above). But in case of failure we are returning an integer (0) here.  Maybe we can try to be more consistent with respect to typing, even though Javascript doesn't care about type-safety?